lib/project_types/extension/models/server_config/development_renderer.rb

```diff
@@ -13,15 +13,16 @@ class DevelopmentRenderer < Base
         ]
 
         property! :name, accepts: VALID_RENDERERS
+        property! :version, accepts: String, default: "latest"
 
         def self.find(type)
           case type.downcase
           when "product_subscription"
-            new(name: "@shopify/admin-ui-extensions")
+            new(name: "@shopify/admin-ui-extensions", version: "^1.0.1")
           when "checkout_ui_extension"
-            new(name: "@shopify/checkout-ui-extensions")
+            new(name: "@shopify/checkout-ui-extensions", version: "~> 0.14.0")
```

Comment:
Should this not be set to compatible version?
```suggestion
            new(name: "@shopify/checkout-ui-extensions", version: "^0.14.0")
```